idd/Energy+.idd.in

```diff
@@ -39847,41 +40047,49 @@ ZoneHVAC:HybridUnitaryHVAC,
+        \note Units for lookup table values should be in C.
         \note If this field is blank, Mode 25 will not be considered for any time step that requires ventilation, heating, cooling, humidification, or dehumidification.
   A244, \field Mode 25 Supply Air Humidity Ratio Lookup Table Name
         \type object-list
         \object-list MultivariateFunctions
         \note Enter the name of the Supply Air Humidity Ratio Lookup Table for Mode 25.
+        \note Units for lookup table values should be in kgWater/kgDryAir.
         \note If this field is blank, Mode 25 will not be considered for any time step that requires ventilation, heating, cooling, humidification, or dehumidification.
   A245, \field Mode 25 System Electric Power Lookup Table Name
         \type object-list
         \object-list MultivariateFunctions
         \note Enter the name of the Electric Power Lookup Table for Mode 25.
+        \note Units for lookup table values should be in W.
         \note If this field is blank, Mode 25 does not consume electricity.
   A246, \field Mode 25 Supply Fan Electric Power Lookup Table Name
         \type object-list
         \object-list MultivariateFunctions
         \note Enter the name of the Supply Fan Electric Power Lookup Table for Mode 25.
+        \note Units for lookup table values should be in W.
         \note If this field is blank, Mode 25 will not be considered for any time step that requires ventilation, heating, cooling, humidification, or dehumidification.
   A247, \field Mode 25 External Static Pressure Lookup Table Name
         \type object-list
         \object-list MultivariateFunctions
         \note Enter the name of the External Static Pressure Lookup Table for Mode 25.
+        \note Units for lookup table values should be in Pa.
         \note If this field is blank, external static pressure will not be reported for Mode 25.
   A248, \field Mode 25 System Second Fuel Consumption Lookup Table Name
         \type object-list
         \object-list MultivariateFunctions
         \note Enter the name of the System Second Fuel Consumption Lookup Table for Mode 25.
+        \note Units for lookup table values should be in W.
         \note If this field is blank, Mode 25 does not consume a second fuel.
   A249, \field Mode 25 System Third Fuel Consumption Lookup Table Name
         \type object-list
         \object-list MultivariateFunctions
         \note Enter the name of the System Third Fuel Consumption Lookup Table for Mode 25.
+        \note Units for lookup table values should be in W.
         \note If this field is blank, Mode 25 does not consume a third fuel.
   A250, \field Mode 25 System Water Use Lookup Table Name
         \type object-list
         \object-list MultivariateFunctions
         \note Enter the name of the System Water Use Lookup Table for Mode 25.
+        \note Units for lookup table values should be in kg/s.
```

Comment:
The number of lookup table fields here is quite ridiculous.  But adding these notes are a good addition that doesn't affect functionality at all, and can only help people get the right inputs in there.